api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/AddressTestDataFactory.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.dhsc.htbhf.claimant.testsupport;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import uk.gov.dhsc.htbhf.claimant.entity.Address;
+
+/**
+ * Test data factory for {@link uk.gov.dhsc.htbhf.claimant.entity.Address} objects.
+ */
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
```

Comment:
Pointless??

Reply:
I really do think this is pointless by the way...

Reply:
We can remove it as a pmd rule. @YetAnotherMatt are you ok with that?

Reply:
Yes, lets remove the PMD rule in test